### PR TITLE
Add validator operator key details

### PR DESCRIPTION
**Pull request by humanalgorithm**
(Merge fork from public awesome)
Adding details of how to view your validator operator key that is required in a step in this doc

### CODE REVIEW

File: docs/setting-up-a-validator.md [hunk 1]
```diff
@@ -81,6 +81,25 @@ Or if your key is stored in a ledger device.
 gravity keys add <my validator key name> --ledger
 ```
 
+## Get Validator Operator Key
+You will need to reference the address of the wallet you intend to use for your validator operator. </br>
+If you already have this wallet address skip this step, otherwise the address can be found by following the below.
+
+The `gravity keys add <validator key name>` generates a validator operator key. </br>
+You can view it with the following command:
```
Comment (jkilpatr) [inline docs/setting-up-a-validator.md:89]: The above second "Add your validator key" addresses some of this same info, having both of these blocks in here at once will be confusing, so their info needs to be integrated. Ideally we can also tie this into the explanation of the other keys required to operate gravity.

**Review by jkilpatr — CHANGES_REQUESTED** [review]

File: docs/setting-up-a-validator.md (hunk 1 above)
Comment (humanalgorithm) [reply docs/setting-up-a-validator.md]: @jkilpatr I collapsed this all under add your validator key as an optional step to be more succinct

**Comment by humanalgorithm** [comment]
@jkilpatr all comments resolved and linter fixed

File: docs/setting-up-a-validator.md (hunk 1 above)
Comment (jkilpatr) [reply docs/setting-up-a-validator.md]: This is compact enough to be good. 👍🏻

**Pull request merged**